docs/release_notes/0.2.0.md

```diff
@@ -0,0 +1,85 @@
+# Release 0.2.0
+
+## Announcement
+
+This release signifies the end of the initial development cycle of `eksctl`.
+We have built many new features since the first minor release (0.1.0)
+in August 2018, and we are switching to a new release cadence now.
+Albeit we have originally planned to do a number of things in 0.2.0,
+the plan has evolved. We will start publishing short-term roadmaps
```

Comment:
I want to put together a roadmap similar to the AWS containers roadmap. So can we say something like: 

We will start publishing our roadmap in a similar form to the [AWS containers roadmap](https://github.com/aws/containers-roadmap/projects/1).

Reply:
That sounds good.